README.md

```diff
@@ -141,14 +141,33 @@ If you only want to download `fastq` and `fastq.gz` files you can run postman wi
 java -jar postman.jar -s .fastq,.fastq.gz
 ```
 
+### How to filter by regular expression
+Both the `download` and the `list` command allow you to filter files using [a regular expression](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html). 
+When filtering with a regular expression pattern, the path of the file as well as the filename are matched against the pattern. Only matching files are included in the output.
+
+If you only want to download files containing `test` in their path or in their name, you can run postman with
+```bash
+java -jar postman.jar --pattern ".*test.*"
```

Comment:
i think that is not a valid command line example. Please provide a complete example